.github/workflows/ci.yaml

```diff
@@ -1,19 +1,17 @@
+---
 name: CI
-
-on:
+'on':
```

Comment:
Hm don't love this, realize you have the issue with ruby dumping this to `true:` but think a little `sed -i 's/true:/on:/g` would be better (this is what we've done before with blaster outer changes to ci.yaml)

Reply:
Yeah, I don't love it either, I'm surprised they used `on` as a key.

I didn't see any other blaster outer scripts updating ci.yaml

Reply:
I didn't create a PR for it but I still have it locally, basically just:
```
gh_actions_ci_yaml = gh_actions_ci.to_yaml
gh_actions_ci_yaml.gsub!(/^true:/, "on:") # YAML replaces on: with true:

File.write(gh_actions_ci_path, gh_actions_ci_yaml)
puts "Wrote updated ci.yml at: #{gh_actions_ci_path}"
```